packages/anthropic/src/tool/bash_20250124.ts

```diff
@@ -15,7 +15,7 @@ export const bash_20250124 = createProviderDefinedToolFactory<
   },
   {}
 >({
-  id: 'anthropic.bashTool_20250124',
+  id: 'anthropic.bash_20250124',
```

Comment:
This was weird and threw me for a loop. According to [Anthropic documentation](https://docs.anthropic.com/en/docs/agents-and-tools/tool-use/bash-tool#tool-versions) it's actually `bash_20250124`, not `bashTool`, HOWEVER, [this quickstart from Anthropic](https://github.com/anthropics/anthropic-quickstarts/blob/main/computer-use-demo/computer_use_demo/tools/bash.py?ref=musings.yasyf.com#L104) shows it as `bashTool`.

Setting it to `bash` seems to work for both, and the documentation shows it that way, so I'm confident in this change.